backend/src/main/java/com/benine/backend/cameracontrol/FocusFunctions.java

```diff
@@ -0,0 +1,50 @@
+package com.benine.backend.cameracontrol;
+
+/**
+ * Interface for Focus functions of camera's.
+ * @author Bryan
+ */
+public interface FocusFunctions {
```

Comment:
Why not simply call this focus? Or focus controller. Functions is maybe a bit vague in my opinion.